seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -146,6 +146,15 @@ bnb,biswap,1,2022-11-22,0x9050a1e2f7aa392e39195b5b22382d595e94104b72699655b198c4
 bnb,biswap,1,2022-11-22,0xe2efd633720bd29322da7e39857e98b3c9904a1c55f9b3995441160983e06296,119,0xe9e7cea3dedca5984780bafc599bd69add087d56,43.898605549510836795,0x563ca064e41f3b5d80adeecfe49ab375fd7afbef,2756.0281
 bnb,biswap,1,2022-11-22,0x0b9ba3ee4a0b3f4db2baba70be527f5a50aea77423cd921106ec9f0ac8dcf169,115,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.018248512111984361,0xe9e7cea3dedca5984780bafc599bd69add087d56,4.837104443141354752 
 bnb,biswap,1,2022-11-22,0xaf03946d9b98e5d6e841bb50555208801e48d773a8dfdf1ecc203d943c27c603,350,0x2170ed0880ac9a755fd29b2688956bd959f933f8,6.247796965743473757,0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c,0.43849
+bnb,sushiswap,1,2022-11-22,0x683b4299466b4fec3fac2859e2dd27215f69879b31a7384593b46aef71952ed5,373,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,6571.818499746365907562,0xe9e7cea3dedca5984780bafc599bd69add087d56,20
+bnb,sushiswap,1,2022-11-22,0xa572f3ea4bcc491c3d6a3b54f2cd59c776238c3187c1ee7975d9635f8a55d129,260,0xe9e7cea3dedca5984780bafc599bd69add087d56,30.330984668706979348,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,10000
+bnb,sushiswap,1,2022-11-22,0xfac7881f1461c9c56a11f68647385ea27c4d66849f646d5f377e558014652df2,536,0xe9e7cea3dedca5984780bafc599bd69add087d56,0.003061259047161221,0xa0ed3c520dc0632657ad2eaaf19e26c4fd431a84,1
+bnb,sushiswap,1,2023-01-17,0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab,186,0xe9e7cea3dedca5984780bafc599bd69add087d56,2.487085655648097925,0xc598275452fa319d75ee5f176fd3b8384925b425,177.678
```

Comment:
test_description | result (model) | expected (seed) | block_date | blockchain | project | version | tx_hash | evt_index
-- | -- | -- | -- | -- | -- | -- | -- | --
matched records count | 0 | 1 | 2022-11-22 00:00 | bnb | sushiswap | 1 | 0x683b4299466b4fec3fac2859e2dd27215f69879b31a7384593b46aef71952ed5 | 373
matched records count | 0 | 1 | 2022-11-22 00:00 | bnb | sushiswap | 1 | 0xa572f3ea4bcc491c3d6a3b54f2cd59c776238c3187c1ee7975d9635f8a55d129 | 260
matched records count | 0 | 1 | 2022-11-22 00:00 | bnb | sushiswap | 1 | 0xfac7881f1461c9c56a11f68647385ea27c4d66849f646d5f377e558014652df2 | 536
matched records count | 0 | 1 | 2023-01-17 00:00 | bnb | sushiswap | 1 | 0x5169181339080e2d56c13c0a075dac19f19e3f8300a8c068e8f4519bf424e7ab | 186

Reply:
These 4 records are not shown in the model query results.
https://dune.com/queries/1908356

Reply:
I think these trades are by `sushi_bnb.UniswapV2Router02`, please have a look in Dune V2 decoded tables.